src/components/Login/Sign-in/Sign-in.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { auth, signInWithGoogle } from '../../../firebase/firebase.utils';
+import Input from '../../shared/Input/Input';
+import Button from '../../shared/Button/Button';
+import './Sign-in.scss';
+
+const SignIn = () => {
+  const [userCredentials, setUserCredentials] = useState({
+    // displayName: '',
+    email: '',
+    password: '',
+    // confirmPassword: '',
+  });
+
+  const { email, password } = userCredentials;
+
+  const handleSumbit = async (event) => {
+    event.preventDefault();
+
+    try {
+      await auth.signInWithEmailAndPassword(email, password);
+      setUserCredentials({ ...userCredentials, email: '', password: '' });
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  const handleChange = (event) => {
+    const { name, value } = event.target;
+
+    setUserCredentials({ ...userCredentials, [name]: value });
+  };
+
+  return (
+    <div className='sign-in'>
+      <h2>It great to have you back on your next training session</h2>
+      <p>Sign in and let's do this</p>
```

Comment:
tutaj mozesz wrzucic bema dla treningu i spojnosci:

sign-in

&__greeting
&__text